### DISCUSSION TITLE
Sunflower Land Music

### DISCUSSION THREAD

**Discussion by Rofrtd** [sunflower-land/sunflower-land#1788] [Art]
**It's time** to submit your own music to Sunflower Land 🎸🎹🎵

We are looking for musicians to add their music onto Sunflower Land.

We all know that great music make games unforgettable, you'll never forget the songs and sound effects from the games you used to play as a kid. 😉

We are looking for music/ideas for a range of different Sunflower Land islands and themes! 
Feel free to do set submissions or one-off ideas ⭐

## Requirements

* As Github does not support MP3 files, you'll need to setup an account on SoundCloud and submit it there.
* Original pieces of work (you **HAVE** to own the rights for the song submitted)

By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your art can be used freely by the project and all community members

When submitting music, please use the following format:

**FarmID: 2**
**Description: This song is called Black Oak and the idea is to be used on the spooky island or during a Halloween event.**
**Link: https://soundcloud.com/romy-furtado/black-oak**

## Timeline

We will be selecting a few songs for the Land Expansion launch but this is an open ended discussion. We are always looking for community submissions - once you have submitted your music pop over to discord and share it with the community to grab the team's attention.

**Comment by RadishPies** [comment]
Farm ID: 3733
Description: This song is called Reap and Sow and I thought it would make a good generic song for the main page playlist.
Link: https://soundcloud.com/radishpies/reap-and-sow

**Comment by RadishPies** [comment]
Farm ID: 3733
Description: This song is called Farming Along because it feels like a great song to listen to when you are working and being productive. Not sure where it could be best used tough.
Link: https://soundcloud.com/radishpies/farming-along

**Comment by RadishPies** [comment]
Farm ID: 3733
Description: This song is called Royal Farms simply because I thought it would be a good fit for the Sunflorea Kingdom project.
Link: https://soundcloud.com/radishpies/royal-farms

**Reply by Rofrtd** [reply]
I love this one 👏🏼

**Comment by digital-farms** [comment]
**Farm ID:** 21637

**Description:** With this track I want to add some rhythm and energy to the game! The track doesn't sound like the usual pixel style, and I think it's very appropriate for Stone Haven Island. All sounds and drums are synthesized by me. The cover artwork is also drawn by me. I'm interested in your opinion!

**Link:** https://soundcloud.com/murrkotov/kotob-crystal-mine

**Comment by kharbush** [comment]
FarmID: 61570
Description: This song is called Dream , music that has a slow tempo can help calm your mind and help the farmer in his journey. i hope u guys  like it <3 <3 !
Link: https://soundcloud.com/goblin-potion/dream

**Reply by LeShykun** [reply]
I LOVE IT!!

**Comment by ctagil** [comment]
FarmID: 2808
Description: I believe this soundtrack will be fine at goblin village or war events i just created and wanna share with you.
Link: https://soundcloud.com/taylan-a-l/goblins-rising

**Comment by celpower** [comment]
Farm 129389
Description: I imagine this song being used on the new are you guys were building during the hackathon
Link: https://on.soundcloud.com/cu9va

**Comment by celpower** [comment]
Farm 129389
Description: I imagine this song being used for a Halloween event or some scary scenario
Link: https://on.soundcloud.com/v65xp
